posts/faq/custom-ui.md

```diff
@@ -4,10 +4,10 @@ title: How do I specify a web-UI to use?
 
 If the files for the UI you'd like to use are stored locally on your computer, start lbry with the `--ui` flag:
 
-    lbrynet-daemon --ui=/full/path/to/ui/files/root/folder
+    `lbrynet-daemon --ui=/full/path/to/ui/files/root/folder`
```

Comment:
@kimihiro64 when text is indented via 4 (or more) spaces, its in a code block. there's no need to wrap that text in ticks. in fact, those ticks will show up as characters in the code.